router.js

```diff
@@ -19,7 +19,8 @@ const Referrer = host => Header('referrer', host.toLowerCase())
 const Path = regExp => req => {
   const url = new URL(req.url)
   const path = url.pathname
-  return path.match(regExp) && path.match(regExp)[0] === path
+  const match = path.match(regExp) | []
```

Comment:
Should this be `||`?

Reply:
:astonished: I mistyped it when copying the change from my TypeScript version. Should have used copy/paste! :laughing: This sort of bug is exactly why I write workers in TypeScript. https://github.com/13rac1/cloudflare-ts-worker-template-router/blob/ce9df454/src/router.ts#L30 It's such a good example, I wish I'd done it intentionally. Fixed.